src/Dapper.Contrib/SqlMapperExtensions.Async.cs

```diff
@@ -30,12 +30,12 @@ public static partial class SqlMapperExtensions
                 var key = GetSingleKey<T>(nameof(GetAsync));
                 var name = GetTableName(type);
 
-                sql = $"SELECT * FROM {name} WHERE {key.Name} = @id";
+                sql = $"SELECT * FROM {name} WHERE {key.Name} = {GetParameterPrefixQuery()}id";
                 GetQueries[type.TypeHandle] = sql;
             }
 
             var dynParams = new DynamicParameters();
-            dynParams.Add("@id", id);
+            dynParams.Add($"{GetParameterPrefixParams()}id", id);

```

Comment:
AFAIK, we dont need to emphasize this prefix in the Dynamic parameter for Dapper. I have been using Dapper for MS SQL Server, Oracle and postgres. They have `@` and `:` as param prefixes. It is only needed in the query but when we add the dynamicParam, it can simply take the name. 

Like,
PostGres:
`
var query = "select * from schemaName.TableName src where  src.BatchRequestNumber = :batchNum";
queryParams.Add("batchNum", batchRequestNumber);
`

Oracle:
`
var query = "select * from schemaName.TableName src where  src.BatchRequestNumber = :batchNum";
queryParams.Add("batchNum", batchRequestNumber);
`

MS SQL:
`
var query = "select * from schemaName.TableName src where  src.BatchRequestNumber = @batchNum";
queryParams.Add("batchNum", batchRequestNumber);
`

Reply:
Are you suggesting to completely remove the prefix on the dynamic parameter (`dynParams.Add("id", id);`) ?

I'm afraid that might introduce some bug or breaking change on some database or provider other than those you mentioned...

Reply:
What are the database(s) it supports currently? Where can I find that information? I agree at this point without knowing what other DBs that this library is setup to support. 

My suggestion is just an option to consider on the Dynamicparameters but having them is an extra step of guarantee. I tested once again on my code where I am using Dapper against Oracle, MS SQL and Postgres.